evap/staff/views.py

```diff
@@ -896,19 +876,15 @@ def user_edit(request, user_id):
 
 
 @staff_required
-def user_delete(request, user_id):
+def user_delete(request):
+    user_id = request.POST.get("user_id")
     user = get_object_or_404(UserProfile, id=user_id)
 
     if user.can_staff_delete:
-        if request.method == 'POST':
-            user.delete()
-            messages.success(request, _("Successfully deleted user."))
-            return redirect('staff:user_index')
-        else:
-            return render(request, "staff_user_delete.html", dict(user_to_delete=user))
+        user.delete()
+        return HttpResponse() # 200 OK
     else:
-        messages.warning(request, _("The user '%s' cannot be deleted, because he lectures courses.") % user.full_name)
```

Comment:
there really was no good reason we had these error messages or was there? users could only see this by manually typing URLs right?